integrations/valkey/pyproject.toml

```diff
@@ -0,0 +1,181 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "valkey-haystack"
+dynamic = ["version"]
+description = ''
+readme = "README.md"
+requires-python = ">=3.9,<3.13"
+license = "Apache-2.0"
+keywords = []
+authors = [{ name = "John Doe", email = "jd@example.com" }]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["haystack-ai>=2.11.0", "valkey-glide>=2.1.0", "valkey-glide-sync>=2.1.0"]
+
+[dependency-groups]
+test = [
+    "pytest>=7.4",
+    "pytest-asyncio>=0.23",
+    "pytest-cov",
+    "pytest-rerunfailures",
+    "mypy",
+]
+examples = [
+    "markdown-it-py",
+    "mdit_plain",
+    "haystack-ai>=2.11.0",
+    "numpy<2.0.0",
+    "torch==2.2.2; sys_platform == 'darwin' and platform_machine == 'x86_64'",
+    "torch>=2.0.0; sys_platform != 'darwin' or platform_machine != 'x86_64'",
+    "sentence-transformers>=5.0.0",
+]
```

Comment:
We don't add dependencies for examples in the `pyproject.toml`. So let's remove it from here, you can make a comment in the example files if there are extra deps to be installed